pkg/reporter/defect_dojo.go

```diff
@@ -0,0 +1,152 @@
+package reporter
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/safedep/vet/pkg/analyzer"
+	"github.com/safedep/vet/pkg/common/logger"
+	"github.com/safedep/vet/pkg/models"
+	"github.com/safedep/vet/pkg/policy"
+)
+
+// DefectDojo accepts findings in SARIF report format. We'll use sarfBuilder
+// to generate the SARIF report and post it to DefectDojo.
+
+const DefaultDefectDojoHostUrl = "http://localhost:8080"
```

Comment:
Not sure if this is a good default. Mostly people will run `vet` in CI and DD will be hosted remotely. In this case I think the `DefectDojoHostUrl` is a required config and we can't really fallback to a default.

Reply:
yep, makes sense, will update